tests/integration.rs

```diff
@@ -65,6 +65,10 @@ fn integration_test() {
         .expect("unable to run clippy");
 
     let stderr = String::from_utf8_lossy(&output.stderr);
+
+    // debug:
+    eprintln!("{stderr}");
```

Comment:
Is this actually a leftover for debugging, or do you want to have this in the tests? From the added `--show-output` flag, I assume this is left here intentionally.

I think this could be quite useful, just want to make sure it wasn't forgotten about.

Reply:
No, this makes us print the clippy command stderr, but it would still be hidden by the testing harness by default, which is why we also need the other flag.

I think its nice to be verbose by default here so we can see what the problem is right away,in case there are any.

Reply:
I thought so. Makes sense, let's ship it :shipit: